sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -146,9 +511,99 @@ public Contact collectContactData() {
         .contactCategory(
             webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(CONTACT_CATEGORY_OPTIONS))
         .relationshipWithCase(
-            webDriverHelpers.getValueFromWebElement(RELATIONSHIP_WITH_CASE_COMBOBOX))
+            webDriverHelpers.getValueFromWebElement(RELATIONSHIP_WITH_CASE_COMBOBOX_INPUT))
+        .descriptionOfHowContactTookPlace(
+            webDriverHelpers.getValueFromWebElement(DESCRIPTION_OF_HOW_CONTACT_TOOK_PLACE_INPUT))
+        .uuid(webDriverHelpers.getValueFromWebElement(UUID_INPUT))
+        .build();
+  }
+
+  public Contact collectContactDataAfterEdit() {
+    webDriverHelpers.waitForPageLoaded();
+    String collectedDateOfReport = webDriverHelpers.getValueFromWebElement(REPORT_DATE);
+    String classification =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(CONTACT_CLASSIFICATION);
+    LocalDate parsedDateOfReport = LocalDate.parse(collectedDateOfReport, formatter);
+    LocalDate parsedDateOfFirst =
+        LocalDate.parse(webDriverHelpers.getValueFromWebElement(FIRST_DAY_CONTACT_DATE), formatter);
+    // LocalDate parsedDateOfFollowUp =
+    //    LocalDate.parse(webDriverHelpers.getValueFromWebElement(FOLLOW_UP_UNTIL_DATE), formatter);
```

Comment:
Please remove the commented code.

Reply:
commented code is required to run when the issue tagged will be fixed. see the feature file.